CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+# Contribution Guidelines
+
+If you are interested in contributing to Substrate Documentation, please follow these contribution 
+guidelines.
+
+* [PR Checklist](#pr-checklist)
+* [Documentation Standards](#documentation-standards)
+* [Documentation Style](#documentation-style)
+
+## PR Checklist
+
```

Comment:
If this is the PR checklist, shouldn't it be rather [be a PR template](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)?

Reply:
Actually now it makes sense. We had this discussion when we first added this but decided against it since it doesn't apply to recipes/tutorials/etc. But now that it's just the KB, makes sense.